### PR TITLE
return Json error messages

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/secure/RapidBoard.jspa?rapidView=831&projectKey=BPS&view=detail&selectedIssue=BPS-783

### Change description ###

return JSON error response

**Does this PR introduce a breaking change?** (check one with "x")

[ ] Yes
[x] No

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/OcrValidationTest.java [hunk 1]
```diff
@@ -130,7 +132,10 @@ void should_return_401_when_service_auth_header_is_missing() throws Throwable {
             post("/forms/PERSONAL/validate-ocr")
                 .contentType(MediaType.APPLICATION_JSON)
                 .content(content))
-            .andExpect(status().isUnauthorized());
+            .andExpect(status().isUnauthorized())
+            .andExpect(jsonPath("$.error").value("Provided S2S token is missing or invalid"))
+            .andExpect(content().string("{\"error\":\"Provided S2S token is missing or invalid\"}"));
```
Comment (rkondratowicz) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/OcrValidationTest.java]: check in line 137 covers 136

File: src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/model/out/ApiError.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    private final String error;
+
+    public ApiError(String error) {
+        this.error = error;
+    }
+
+    public String getError() {
+        return error;
+    }
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/model/out/ApiError.java]: we usually use public final fields and getters is not needed then.

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/OcrValidationTest.java (hunk 1 above)
Comment (mdayican) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/OcrValidationTest.java]: removed

File: src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/ResponseExceptionHandler.java [hunk 3]
```diff
@@ -25,29 +26,30 @@
 public class ResponseExceptionHandler extends ResponseEntityExceptionHandler {
 
     private static final Logger log = LoggerFactory.getLogger(ResponseExceptionHandler.class);
+    private static final ApiError API_ERROR_FORBIDDEN = new ApiError("S2S token is not authorized to use the service");
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/ResponseExceptionHandler.java]: Only single use of this private field

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/ResponseExceptionHandler.java]: what do you mean?

**Comment by mdayican** [comment]
will commit from a new branch.

**Pull request closed without merging**